docs/podman-run.1.md

```diff
@@ -655,6 +655,22 @@ Set the UTS mode for the container
 
 **NOTE**: the host mode gives the container access to changing the host's hostname and is therefore considered insecure.
 
+**−−mount**[=*type=<bind|tmpfs>,<src|source>=HOST−DIR,<target|dst,destination>=CONTAINER−DIR[:OPTIONS]*]
+The --mount flag allows you to mount host-directories and tmpfs mounts in a container.
+The --mount flag supports most options that are supported by the -v|--volume flag, but uses a different syntax.
+
+The type must be bind or tmpfs. *Note: the tmpfs mounted directory will not be retained after the container stops.*
+
+The source can be specified as "source" or "src" & target can be specified as "target","dst" or "destination".
+The CONTAINER−DIR must be an absolute path such as /src/docs. The HOST−DIR must be an absolute path as well.
+podman bind −mounts the HOST−DIR to the path that is specified.
+For example, if src=/foo,target=/foo is provided, podman creates a bind −mount. Multiple --mount options can be
```

Comment:
unwanted space between "bind" and " -mount"?

Reply:
I'm on the fence.  Your call.  Should we fluff up the end of this first sentence a bit?  Maybe ", podman creates a bind-mount from the src directory on the host to the target directory on the container."

Reply:
Rewrote this section based on docker run --mount description